src/Lorentz/Contracts/Metadata.hs

```diff
@@ -14,7 +14,35 @@ module Lorentz.Contracts.Metadata
 
 import Lorentz
 
-import Lorentz.Contracts.Spec.FA2Interface (TokenId, TokenMetadata(..), theTokenId)
+import Fmt (Buildable(..), (+|), (|+))
+
+import Lorentz.Contracts.Spec.FA2Interface (TokenId, theTokenId)
+
+data TokenMetadata = TokenMetadata
+  { tmTokenId :: TokenId
+  , tmSymbol :: MText
+  , tmName :: MText
+  , tmDecimals :: Natural
+  , tmExtras :: Map MText MText
+  }
+  deriving stock (Eq, Show)
+
+$(customGeneric "TokenMetadata" rightComb)
+
+deriving anyclass instance IsoValue TokenMetadata
+
+instance HasAnnotation TokenMetadata where
+  annOptions = defaultAnnOptions { fieldAnnModifier = dropPrefixThen toSnake }
+
+instance Buildable TokenMetadata where
+  build TokenMetadata {..} =
+    "token_id:" +| tmTokenId |+ ", symbol:" +| tmSymbol |+ ", name:" +|
+    tmName |+ ", decimals:" +| tmDecimals |+ "."
+
+instance TypeHasDoc TokenMetadata where
+  typeDocMdDescription =
+    "Contract's storage holding a big_map with all balances and the operators."
```

Comment:
This seems incorrectly copied.

Reply:
Actually, this is correctly copied, see [this piece of code](https://gitlab.com/morley-framework/morley-ledgers/-/blob/a8ba95b28e2a267de22701cf4d502e7bc07c2c6b/code/morley-ledgers/src/Lorentz/Contracts/Spec/FA2Interface.hs#L175) :joy:. 
However, I agree that it's some nonsense `TypeHasDoc` instance

Reply:
Oh ok lol.

It's such a generic description and I saw a similar instance for something else, so I though it was here by accident.

It's ok than, no problem.